book/content/diving_into_leadership/diving_into_leadership.md

```diff
@@ -0,0 +1,49 @@
+# Personal Experience: Diving into Leadership
+
+_The following blog post was written by The Turing Way contributor [Sarah Gibson](https://www.turing.ac.uk/people/researchers/sarah-gibson)._
+
+Hackathons can be quite intimidating scenarios to put yourself into.
+You're probably meeting a whole load of new people and getting to grips with a new project or programming language.
+On top of that, you probably only have a couple of days or less to produce something to show for your time!
+
+Now imagine if it was your job to lead and coordinate that group of people to work on your own project?
+
+Don't worry! I've been there, and I was intimidated too!
+So let me share some of things I learned the first time I lead a hackathon project.
+
+## Don't just focus on the code
+
+There are lots of different things that go into making a project, beyond the code itself.
+Similarly, there are lots of different people with different skills who may join your hackathon project - not just programmers!
+Good leaders can identify the strengths of their team and specifically where their skills can be put to best use on a project.
+
+Things like documentation are often left until the last moments of a project even though it's just as important, if not more so, than the code itself!
+If you have someone on your team who likes to write, documentation can be kept up-to-date as your codebase changes through the hackathon.
+
+The largest benefit of having a well documented piece of software at the end of the hackathon is that it will be much easier for your team to continue to contribute to the project after the event.
+The last phase of getting a project into a production ready state takes the most time and effort, but luckily you've already documented your scoping process and pathway to getting there! :wink:
+
+## Find your style of leadership
+
+Given this was my first hackathon as a project lead, I found it very beneficial for me to be operating at quite a high-level on the project, rather than taking a deep dive into the code.
```

Comment:
```suggestion
Given this was my first hackathon as a project lead, I found it very beneficial to be operating as an overseer of the project, rather than taking a deep dive into the code.
```

Reply:
is this what you mean?